src/lib/crypto/kimchi_backend/gadgets/foreign_field.ml

```diff
@@ -1194,13 +1170,6 @@ let compute_high_bound (x : Bignum_bigint.t)
   assert (Bignum_bigint.(x_bound_hi < Common.two_to_limb)) ;
   x_bound_hi
 
-(* Perform integer bound addition for all limbs x' = x + f' *)
-let _compute_bound (x : Bignum_bigint.t)
```

Comment:
don't we need it anymore, not even inside the canonical check?

Reply:
There is `check_canonical` now, which does it the other way with `x + 2^264`

Reply:
The `x' = x + f'` was the way my old code computed it using the negated non-native field modulus.  It was just a more convenient way to compute the same thing, since I had the negated modulus instead.